kf_lib_data_ingest/common/pandas_utils.py

```diff
@@ -6,9 +6,81 @@
 import numpy
 import pandas
 
+from kf_lib_data_ingest.common.misc import multisplit
 from kf_lib_data_ingest.common.type_safety import assert_safe_type
 
 
+class Split:
```

Comment:
jw why you didn't do:

```python
class Split:
    def __init__(self, things):
        assert_safe_type(things, list)
        self.things = things

s = Split() 
s.things
```

Reply:
I originally had a different vision for it and this is just leftover. No reason now to not look at things directly.

Reply:
👍